### ISSUE TITLE
Get version from git using setuptools_scm

### ISSUE THREAD

**Issue by KOLANICH** [tqdm/tqdm#722]
- [x] I have visited the [source website], and in particular
  read the [known issues]
- [x] I have searched through the [issue tracker] for duplicates

`setuptools_scm` gets the version from git correctly. The currently used way to get version from git is buggy, for example it breaks on linux.

Depends on #721

**Comment by casperdcl** [comment]
merged into #721
